src/app/containers/PageHandlers/withVariant/getVariantRedirectUrl/index.test.js

```diff
@@ -0,0 +1,321 @@
+import Cookie from 'js-cookie';
+import {
+  getVariant,
+  servicesWithVariants,
+} from '#lib/utilities/variantHandler';
+import { articlePath, frontPagePath } from '#app/routes/regex';
+import getVariantRedirectUrl from './index';
+
+jest.mock('js-cookie', () => ({
```

Comment:
you don't need to mock `js-cookie` because jsdom (which is the default environment in jest) has a cookie jar https://github.com/jsdom/jsdom#cookie-jars

you can test this out in your test by doing:

```js
Cookie.set('cookiename', 'cookievalue');
console.log(document.cookie); // cookiename=cookievalue
```

if you want to clear out cookies after each test then do

```js
  afterEach(() => {
    jest.clearAllMocks();
    document.cookie = '';
  });
```

I think all your tests will still pass if you do the above and remove the `js-cookie` mock

Reply:
Fixed!